crates/ruff_python_formatter/resources/test/fixtures/ruff/docstring_code_examples.py

```diff
@@ -323,3 +344,411 @@ def doctest_invalid_skipped_with_triple_double_in_single_quote_string():
     >>> x        =      '\"\"\"'
     """
     pass
+
+
```

Comment:
I may have missed it, but in case there's no such test. Could we add a test that tests mixing restructured text and markdown exmaples in a single docstring? Including when we have an unclosed markdown test.